src/main/java/babblebot/ui/Ui.java

```diff
@@ -0,0 +1,146 @@
+package babblebot.ui;
+import babblebot.task.Task;
+import babblebot.task.TaskList;
+import java.util.Scanner;
+
+/**
+ *
+ * The Ui class handles all interactions with the user.
+ * It provides methods to display messages, read user input, and show the current task list.
+ */
+public class Ui {
+    private Scanner sc;
+    private String SEPARATOR = "------------------------------------------------------------------";
```

Comment:
I am pretty sure this was meant to be a constant, as seen from your use of SCREAMING_SNAKE_CASE. Adding a `final` keyword to the variable might make that idea more explicit 😄